package.json

```diff
@@ -20,11 +20,11 @@
     "test:external": "node scripts/test-external-project.js",
     "test:external-repos": "npm run test:external external-repos",
     "lint": "run-s lint:ts lint:js",
-    "lint:js": "node_modules/.bin/eslint . -f stylish",
-    "lint:ts": "node_modules/.bin/tslint -t stylish --project .",
+    "lint:js": "node_modules/.bin/eslint --ext .js .",
```

Comment:
can we combine 2 lint commands into 1 ?

Reply:
They are already combined with the "lint fix" command, but do you want me to do it regardless?

Reply:
Oh I mean we can remove `lint:js`, `lint:fix:js`, `lint:ts`, `lint:fix:ts` and use only `lint` and `lint:fix`

Reply:
Ok now I get you, will fix it :)